libr/bin/format/coff/coff_specs.h

```diff
@@ -153,6 +153,14 @@
 #define COFF_REL_AMD64_REL32_4		8
 #define COFF_REL_AMD64_REL32_5		9
 
+#define COFF_REL_ARM_BRANCH24T		20
```

Comment:
i prefer space indents to avoid those problems when the first char of the line is not a tab

Reply:
Every line of the file with `#define` contains tabs. Should I format them all?

Reply:
yep

Reply:
I think it's better to avoid reformatting changes in unrelated PR. It can be done separately.

Reply:
@radare @XVilka  So what is your final decision?

Reply:
@pelijah don't do formatting in this PR. Lets merge it as is.